packaging/installer/UPDATE.md

```diff
@@ -24,28 +24,55 @@ you installed it.** Choose from the following list to see the appropriate update
 
 -   [One-line installer script (`kickstart.sh`)](#one-line-installer-script-kickstartsh)
 -   [`.deb` or `.rpm` packages](#deb-or-rpm-packages)
--   [Pre-built static binary for 64-bit systems
-    (`kickstart-static64.sh`)](#pre-built-static-binary-for-64-bit-systems-kickstart-static64sh)
+-   [Pre-built static binary for 64-bit systems (`kickstart-static64.sh`)](#pre-built-static-binary-for-64-bit-systems-kickstart-static64sh)
 -   [Docker](#docker)
 -   [macOS](#macos)
 -   [Manual installation from Git](#manual-installation-from-git)
 
+#### How to determine which install method you used
+
+First, figure out where your user configuration files for Netdata are, normally this is `/etc/netdata`
+or `/opt/netdata/etc/netdata`. You can determine this from the system you installed by running the following command:
+
+```bash
+curl -sS http://localhost:19999/netdata.conf | grep 'config directory'
```

Comment:
@Ferroin if the dashboard is down this won't work but it's ok

Reply:
Yeah, there’s unfortunately not any good way to do this without involving the dashboard or going through a long list of shell commands. I think it’s safe to assume that most users _probably_ don’t actually need this command, but among those who do they are probably using default settings so it _should_ work in most cases.

Reply:
We could add sentence after this command saying something like, "If the `curl` command failed, try manually navigating to either `/etc/netdata` and `/opt/netdata/etc/netdata` and use `ls` to look for the presence of the `netdata.conf` file."